### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by serverul** [g3w-suite/g3w-suite-docker#113]
### Discussed in https://github.com/g3w-suite/community/discussions/6

<div type='discussions-op-text'>

<sup>Originally posted by **serverul** November 26, 2023</sup>
Hi everybody. 

I am trying to install the app but I receive some errors the the app doesn't function at all.

I've tried with docker version on both my proxmox server and on linode vps. I've tried lxc container and vm as well. Nothing seems to work. On the docker install everything goes well and without errors, all the containers start, only the g3w-suite one is loop restarting, resulting in the app to not function. 

On the raw installation it give me an error when I try to install it with paver. Always the same error:

```sh
 ---> pavement.sync
python g3w-admin/manage.py migrate --noinput

Build failed running pavement.install: Subprocess return code: 1
```

Does anyone have a complete installation guide? I am sure that are some steps that are missing because the issue replicates on other computers and other networks as well. 
Either on docker or raw install. </div>

**Comment by serverul** [comment]
Retried for 'n' time with docker again. Install is going well but service is not working.
 
```

root@webgis:~# docker-compose ps
          Name                         Command                 State                                              Ports
----------------------------------------------------------------------------------------------------------------------------------------------------------------
root_certbot_1              certbot                          Exit 1
root_g3w-suite-consumer_1   /scripts/docker-consumer-e ...   Up
root_g3w-suite_1            /bin/sh -c /scripts/docker ...   Up           8000/tcp
root_nginx_1                /docker-entrypoint.sh ngin ...   Up           0.0.0.0:443->443/tcp,:::443->443/tcp, 80/tcp, 0.0.0.0:8080->8080/tcp,:::8080->8080/tcp
root_postgis_1              /bin/bash /scripts/docker- ...   Restarting
root_redis_1                docker-entrypoint.sh redis ...   Up           6379/tcp

```

**Comment by serverul** [comment]
The install issue presented here was resolved in [#6](https://github.com/g3w-suite/community/discussions/6) 

Now I have other wall to climb. I cannot add a project because I receive an error regarding 

```
[Project error on layers()]-- [Layer error on datasource(Arbori_noi_41ea6fd2_9639_4799_8d41_f3618e8c3784)]-- : Current datasource is /shared-volume/media/temp_uploads/Arbori_noi.gpkg|layername=Arbori_noi
```

I detailied on the disscussion [#112](https://github.com/g3w-suite/community/discussions/6) 
Any Ideas?
